exec/jdbc/src/main/java/org/apache/drill/jdbc/DrillConnection.java

```diff
@@ -213,4 +213,5 @@ void setNetworkTimeout( Executor executor, int milliseconds )
 
   DrillClient getClient();
 
+  boolean useAnsiQuotedIdentifiers() throws SQLException;
```

Comment:
Done.
This is the similar approach like in mySQL - setting quoting identifier character during the connection time.
Since a new RPC request is implemented, it is no longer necessary.